__tests__/article.test.js

```diff
@@ -279,3 +279,24 @@ describe("08 PATCH /api/articles/:article_id", () => {
     });
   });
 });
+
+describe("10 GET /api/articles?topic=*", () => {
+  describe("Functionality", () => {
+    test("when provided with a topic query, the returned Articles should only be of that topic", () => {
+      return request(app)
+        .get("/api/articles?topic=mitch")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles.length).toEqual(12);
+          articles.forEach((article) => {
+            expect(article).toEqual(
+              expect.objectContaining({
+                topic: "mitch",
+              })
+            );
+          });
```

Comment:
I understand the conflict on error handling this one. A few things to have a think on.
- what about a valid topic that _is_ in the database but has no articles (eg paper) what status would that be?
- Why would we have a topic's table if we were to accept any topic query to return an empty array?